MAUI/ImageEditor/Effects.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: Image Effect in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about Image Effect support in Syncfusion .NET MAUI Image Editor (SfImageEditor) control and more.
+platform: maui
+control: SfImageEditor
+documentation: ug
+---
+
+# Image Effect in .NET MAUI Image Editor (SfImageEditor)
+
+Using the image editor control, you can add effects such as Hue, Saturation, Brightness, Contrast, Blur, and Sharpen to the image. These effects can be applied from toolbar or using the ImageEffect method. The [`ApplyImageEffect`] method contains two arguments: [`ImageEffect`] and [`EffectValue`]. The ImageEffect is an Enum, which contains the following effects:

```

Comment:
With the image editor control, you can apply various effects such as Brightness, Hue, Saturation, Contrast, Blur, Opacity, and Sharpen to your image. These effects can be applied using the toolbar or by utilizing the `ImageEffect` method. The `ImageEffect` method consists of two arguments: `ImageEffect` and `EffectValue`. The ImageEffect is an enumeration that includes the following effects:

Reply:
content updated

Reply:
content not updated